search-replace.go

```diff
@@ -155,71 +162,229 @@ func fixLine(line *[]byte, replacements []*Replacement) *[]byte {
 	return line
 }
 
-var serializedStringPrefixRegexp = regexp.MustCompile(`s:(\d+):\\"`)
-
 func fixSerializedContent(line *[]byte, replacements []*Replacement) *[]byte {
-	startIndex := 0
-	for startIndex < len(*line) {
-		Debugf("Start of loop, startIndex: %d\n", startIndex)
-		match := serializedStringPrefixRegexp.FindSubmatchIndex((*line)[startIndex:])
-		if match == nil {
+	index := 0
+
+	var rebuiltLine []byte
+
+	for index < len(*line) {
+		Debugf("Start of loop, index: %d\n", index)
+		linePart := (*line)[index:]
```

Comment:
Once a certain part of the line has been processed, we slice it away to create a new `linePart`